tldr/cli.py

```
@@ -72,6 +72,14 @@ def _get_subprocess_detach_kwargs():
     '.exs': 'elixir',
 }
 
+# Canonical list of supported languages for --lang choices
+LANG_CHOICES = [
+    "auto", "python", "typescript", "javascript", "go", "rust", "java",
+    "c", "cpp", "ruby", "php", "kotlin", "swift", "csharp", "scala",
+    "lua", "luau", "elixir",
+]
+LANG_CHOICES_WITH_ALL = LANG_CHOICES + ["all"]
```

Ruff (0.15.5)

[warning] Consider `[*LANG_CHOICES, "all"]` instead of concatenation

Replace with `[*LANG_CHOICES, "all"]`

(RUF005)